### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by gabrielazanova**
Currently the Register here link redirects to the registration page of GameBus

### CODE REVIEW

File: diabetes-dashboard/src/App.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
     <div id="app">
-        <Header class="header"/>
-        <div class="clearfix"></div>
+        <Header class="header" v-show="$route.path==='/login' ? false : true" ></Header>
+        <div class="clearfix" v-show="$route.path==='/login' ? false : true"></div>
```
Comment (JfmGijsbers) [inline diabetes-dashboard/src/App.vue:4]: Use "v-if" to do this. v-show renders everything and then hides it, while v-if skips rendering completely if the condition is false, thereby increasing performance (https://www.digitalocean.com/community/tutorials/vuejs-conditional-directives)

File: diabetes-dashboard/src/router/router.js [hunk 2]
```diff
@@ -9,6 +9,12 @@ const routes = [
         component: () => import('@/views/Dashboard.vue'),
         props: { param: "optional" },
         name: "dashboard"
+    },
+    {
+        path: '/login',
+        component: () => import('@/views/Login.vue'),
+        props: { param: "optional" },
```
Comment (JfmGijsbers) [inline diabetes-dashboard/src/router/router.js:16]: Remove unused paramameter

File: diabetes-dashboard/src/views/Login.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+              class="customField"
+              label="Email"
+              prepend-inner-icon="mdi-account"
+            ></v-text-field>
+            <v-text-field
+              class="customField"
+              label="Password"
+              prepend-inner-icon="mdi-lock"
+              :append-icon="showPass ? 'mdi-eye' : 'mdi-eye-off'"
+              :type="showPass ? 'text' : 'password'"
```
Comment (JfmGijsbers) [inline diabetes-dashboard/src/views/Login.vue:19]: This is exactly what I meant, nice!

**Review by JfmGijsbers — COMMENTED** [review]
Only some very small changes, looks good!

**Review by nbalasovs — APPROVED** [review]

**Pull request merged**